pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -258,6 +247,17 @@ func (o *GinkgoRunSuiteOptions) Run(suite *TestSuite, junitSuiteName string, mon
 		parallelism = 10
 	}
 
+        restConfig, err := clusterinfo.GetMonitorRESTConfig()
```

Comment:
@Rajalakshmi-Girish two things to address:
1. please move this right after the `DryRun` one, that's sufficient to make it work
2. please make sure to `gofmt` your code, otherwise it'll fail

Reply:
@soltysh Can you please check now?